nikola/plugins/command/import_wordpress.py

```diff
@@ -85,9 +81,16 @@ def install_plugin(site, plugin_name, output_dir=None, show_install_notes=False)
     if plugin_installer.execute(options=options) > 0:
         return False
     # Let the plugin manager find newly installed plugins
-    site.plugin_manager.collectPlugins()
-    # Re-scan for compiler extensions
-    site.compiler_extensions = site._activate_plugins_of_category("CompilerExtension")
+    old_candidates = set(site.plugin_manager.candidates.copy())
```

Comment:
Is the `copy()` necessary here, since you create a new `set()` out of it anyway?